walkthroughs/week-3-server/examples/server-stats/src/main/webapp/script2.js

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019 Google LLC
```

Comment:
This change seems unrelated to the comments functionality, so please move it to a different pull request

Reply:
I'm not sure how to do that in this request, but I will avoid adding unrelated commits in the future